pymeasure/instruments/keithley/keithley6517b.py

```diff
@@ -0,0 +1,343 @@
+#
```

Comment:
Please add docs for this instrument by following the instructions in the Contributing guide:
https://pymeasure.readthedocs.io/en/latest/dev/adding_instruments.html

Reply:
docs have been added.